File: Compiled/fibonacci-recursive/fib.c
#include <stdio.h>
#include <stdlib.h>

int fib(int n) {
    if (n > 1) {
        return fib(n-1) + fib(n-2);
    }
    return n;
}

int main(int argc, const char * argv[]) {
    printf("%d\n", fib(atoi(argv[1])));
    return 0;
}

File: Compiled/fibonacci-recursive/fib.go
package main

import (
	"fmt"
	"os"
	"strconv"
)

func fib(n int) int {
	if n > 1 {
		return fib(n-1) + fib(n-2)
	}
	return n
}

func main() {
	n, _ := strconv.Atoi(os.Args[1])
	fmt.Println(fib(n))
}
